### ISSUE TITLE
Add startup tests for machine word size

### ISSUE THREAD

**Issue by andrewthad** [andrewthad/rotera#9]
The server should fail to start on a 32-bit platform.

**Comment by chessai** [comment]
can we not just add

```
when (sizeOf @Int undefined /= 8) $ do
  throwIO (AssertionFailed "Not on a 64-bit platform.")
```

as the first line of the server?

**Comment by andrewthad** [comment]
That is exactly what we should add.

**Comment by chessai** [comment]
Okay, done

**Comment by chessai** [comment]
Will push when i get out of the bathroom

**Comment by chessai** [comment]
fixed by 870bba099815a281406176c3ad727d44876e9446
